e2e/config.js

```diff
@@ -17,6 +17,7 @@ module.exports = {
 		enterOtherProceedings: 'Enter other proceedings',
 		enterAllocationProposal: 'Enter allocation proposal',
 		attendingHearing: 'Attending hearing',
-		uploadDocuments: 'Upload documents',
+		uploadDocuments: 'Annex documents',
 	},
+	testFile: 'mockFile'
```

Comment:
Lets put that into some logic directory so no test files are in root of the project